### ISSUE TITLE
TOC Generator

### ISSUE THREAD

**Issue by pi0** [unjs/automd#4]

**Comment by cpreston321** [comment]
It seems this is already supported within Github unless you are looking for a universal use case.

Links 
- https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

**Comment by pi0** [comment]
Yes indeed idea is for universal usage like a rendered HTML not particularly github that has it.

**Comment by uncenter** [comment]
> Yes indeed idea is for universal usage like a rendered HTML not particularly github that has it.

Especially so because READMEs are often published to places like NPM that don't have that feature.

**Comment by pi0** [comment]
Now i wish we could have a way to hide it only for GitHub 😆 (there might be a CSS trick just guessing)

**Comment by uncenter** [comment]
I think this would involve maybe adding a heading parser to `omark` that we build on top of here?

**Comment by pi0** [comment]
it can be now implemented with mdbox parser
